### PR TITLE
Use Composer

### PR DESCRIPTION
**Pull request by yguedidi**
Changes proposed in this pull request:

- use Composer for autoloading
- move PHPMailer as a Composer dependency
- maybe more?

How to test the feature manually:

1. just play with the app

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested
- [x] documentation updated

### CODE REVIEW

File: docs/en/admins/02_Prerequisites.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ You need to verify that your server can run FreshRSS before installing it. If yo
 | PHP           | **PHP 7+**              |                         |
 | PHP modules   | Required: libxml, cURL, JSON, PDO_MySQL, PCRE and ctype.<br />Required (32-bit only): GMP <br />Recommended: Zlib, mbstring, iconv, ZipArchive<br />*For the whole modules list see [Dockerfile](https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/Dockerfile-Alpine#L7-L9)* | |
 | Database      | **MySQL 5.5.3+**        | SQLite 3.7.4+, PostgreSQL 9.5+          |
-| Browser       | **Firefox**             | Chrome, Opera, Safari, or Edge          |
```
Comment (Frenzie) [inline docs/en/admins/02_Prerequisites.md:13]: While I agree the utility of this line is debatable, I'd prefer not to see an unrelated change like this in a PR. ;-)

Comment (yguedidi) [reply docs/en/admins/02_Prerequisites.md:13]: oups, didn't realized it got removed, sorry

Comment (yguedidi) [reply docs/en/admins/02_Prerequisites.md:13]: fixed

**Comment by Alkarex** [comment]
Thanks for the draft (not working though). Closing for now as responded on https://github.com/FreshRSS/FreshRSS/discussions/4317#discussioncomment-2586621 but might become handy in the future

**Comment by yguedidi** [comment]
no problem @Alkarex ! I was prepared that it may certainly end up being closed :)

> not working though

what was the issue?

**Comment by Alkarex** [comment]
> what was the issue?

From memory of quick test: the code stopped at an unmet dependency (`FreshRSS_Context`). But from the diff, several other things would likely fail, such as some of the removed require/include (e.g. `require_once(LIB_PATH . '/favicons.php');`), and our Docker builds.

**Comment by yguedidi** [comment]
@Alkarex really weird! on my local I managed to make it work perfectly!
have you run `composer install`? have you rebuild the Docker container after?
